misc_analysis/viz_Pa_simulated_experiment_compendium.ipynb

```diff
@@ -0,0 +1,2156 @@
+       "\n",
+       "Gene_symbol                      PA5563    PA5564    PA5565    PA5566  \\\n",
+       "05_PA14000-4-2_5-10-07_S2.CEL  0.790965  0.893249  0.789939  0.164157   \n",
+       "54375-4-05.CEL                 0.733186  0.639074  0.681204  0.110301   \n",
+       "AKGlu_plus_nt_7-8-09_s1.CEL    0.773422  0.791118  0.931585  0.172570   \n",
+       "anaerobic_NO3_1.CEL            0.593955  0.527203  0.706524  0.205510   \n",
+       "anaerobic_NO3_2.CEL            0.571460  0.473054  0.669643  0.155548   \n",
+       "\n",
+       "Gene_symbol                      PA5567    PA5568    PA5569    PA5570  \\\n",
+       "05_PA14000-4-2_5-10-07_S2.CEL  0.970470  0.887472  0.900484  0.880012   \n",
+       "54375-4-05.CEL                 0.619554  0.747656  0.749893  0.805374   \n",
+       "AKGlu_plus_nt_7-8-09_s1.CEL    0.797148  0.753785  0.856253  0.811099   \n",
+       "anaerobic_NO3_1.CEL            0.504767  0.105662  0.363409  0.544780   \n",
+       "anaerobic_NO3_2.CEL            0.562927  0.049738  0.388931  0.548814   \n",
+       "\n",
+       "Gene_symbol                    experiment_id  \n",
+       "05_PA14000-4-2_5-10-07_S2.CEL    E-MEXP-2593  \n",
+       "54375-4-05.CEL                            NA  \n",
+       "AKGlu_plus_nt_7-8-09_s1.CEL      E-MEXP-2867  \n",
+       "anaerobic_NO3_1.CEL                       NA  \n",
+       "anaerobic_NO3_2.CEL                       NA  \n",
+       "\n",
+       "[5 rows x 5550 columns]"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Add experiment id to original gene expression data\n",
+    "sample_ids = list(normalized_data.index)\n",
+    "normalized_data_label = normalized_data.copy()\n",
+    "for sample_id in sample_ids:\n",
+    "    if sample_id in list(metadata.index):\n",
+    "        if metadata.loc[sample_id].ndim == 1:\n",
+    "            normalized_data_label.loc[sample_id,'experiment_id'] = metadata.loc[sample_id,'experiment']\n",
+    "        else:\n",
+    "            normalized_data_label.loc[sample_id,'experiment_id'] = 'NA'\n",
+    "    else:\n",
+    "        normalized_data_label.loc[sample_id,'experiment_id'] = 'NA'\n",
+    "\n",
+    "normalized_data_label.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualize data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Select example experiments\n",
+    "example_id_sim = \"E-GEOD-51409_173\"\n",
+    "example_id = \"E-GEOD-51409\"\n",
+    "#example_id_sim = \"E-GEOD-52445_58\"\n",
+    "#example_id = \"E-GEOD-52445\"\n",
+    "#example_id_sim = \"E-GEOD-18594_2\"\n",
+    "#example_id = \"E-GEOD-18594\"\n",
+    "#example_id_sim = \"E-MEXP-2606_0\"\n",
+    "#example_id = \"E-MEXP-2606\""
```

Comment:
I was trying to find an experiment with enough within experiment structure (i.e. 2 clusters of samples) to demonstrate the conservation of sample-sample relationships.  This is hard to find presumably because the data is in such high dimensional space that that kind of structure is hard to detect